src/relax/op/nn/nn.cc

```diff
@@ -191,6 +205,28 @@ StructInfo InferStructInfoBatchNorm(const Call& call, const BlockBuilder& ctx) {
   }
 }
 
+InferLayoutOutput InferLayoutBatchNorm(const Call& call,
+                                       const Map<String, Array<String>>& desired_layouts,
+                                       VarLayoutMap var_layout_map) {
+  ICHECK(NoDesiredLayout(call, desired_layouts));
+  std::vector<NLayout> initial_layouts;
+  for (size_t i = 0; i < 5; ++i) {
+    const auto* tensor_sinfo = GetStructInfoAs<TensorStructInfoNode>(call->args[i]);
+    ICHECK(tensor_sinfo != nullptr) << "Invalid Call";
```

Comment:
For op that have two or more arguments with all the args being Tensors, we have `GetInputTensorStructInfo` to fetch their TensorStructInfo https://github.com/mlc-ai/relax/blob/63813db4c883d0dc35d14f48dd0bd3dac744fc9a/src/relax/op/nn/nn.cc#L192-L193